docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -574,31 +577,84 @@ V&V Checks:
     - 100,000
     - Locations include Pakistan, Nigeria, and Ethiopia. 
       10 seeds * 10,000 simulants = 100,000 total population.
+    - Baseline only
   * - 3.1
     - Wave I Neonatal disorders V&V with correct LBWSG distribution
     - Complete
     - N/A
     - 10
     - 100,000
+    - Baseline only
     - Locations include Pakistan, Nigeria, and Ethiopia. 10 seeds * 10,000 simulants = 100,000 total population.
   * - 3.2
     - Wave I Neonatal disorders V&V with LBWSG component removed
     - Complete
     - N/A
     - 10
     - 100,000
+    - Baseline only
     - Locations include Pakistan, Nigeria, and Ethiopia. 10 seeds * 10,000 simulants = 100,000 total population.
   * - 3.3
     - Wave I Neonatal disorders V&V with early NN observer bugfix
     - Complete
     - N/A
     - 10
     - 100,000
+    - Baseline only
+    - Locations include Pakistan, Nigeria, and Ethiopia. 10 seeds * 10,000 simulants = 100,000 total population.
```

Comment:
I like to let the engineers decide how to balance seeds and simulants-per-seed --- if 5 seeds * 20,000 simulants is faster, no need to lock in 10 * 10,000.  
```suggestion
    - Locations include Pakistan, Nigeria, and Ethiopia.  100,000 total population (e.g. 10 seeds * 10,000 simulants).
```